lib/ansible/modules/tacp/tacp_instance.py

```diff
@@ -87,64 +89,108 @@
 
 def run_module():
```

Comment:
You have got to refactor this. These functions are too long and too messy.
If it is a constant, do not place it in the function.
Do not have functions defined in your functions.

I recommend that you split larger regions in these large functions into separate functions. Like `validate_boot_order_with_template`, `create_boot_order_payload`, etc. Then use these functions to improve the readability of the code.